src/ocaml_lsp.ml

```diff
@@ -222,6 +244,7 @@ let is_version_up_to_date t ocaml_v =
             else Error (`Newer_available (Some v, available.(last)))))
   in
   Result.map_error res ~f:(fun err ->
+    let _ = suggest_to_upgrade_ocaml_lsp_server () in
```

Comment:
```suggestion
```

Hum I now realize that this introduce a bad circle: `suggest` calls `upgrade_ocaml_lsp_server` which calls `is_version_up_to_date` :-) 

Let's keep it simple for now, and suggest to upgrade after calls to `is_version_up_to_date` when it makes sense.